CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * Intel Gaudi software has been updated to 1.17.1 with Python 3.11 and
   Torch 2.3.1 support.
 * `--legacy` has been removed and replaced with `--pipeline=simple` in `ilab model train`
+* llama-cpp-python has been updated to 0.3.1, which dropped some accelerators (cuBLAS, clBLAS) and added others (OpenBLAS).
```

Comment:
This should be under a new `0.20.0` section - `0.19.0` has already been released